diff -ruN working/file1 initial/file1
--- working/file1	2020-11-18 16:05:13.664910742 +0000
+++ initial/file1	2020-11-18 16:06:12.012873364 +0000
@@ -1,3 +1,2 @@
 dkmsdmsdmkc
-mdscsdkcs
-klfrldc
\ No newline at end of file
+mdscsdkcs
\ No newline at end of file
diff -ruN working/file2 initial/file2
--- working/file2	2020-11-18 16:05:16.344907657 +0000
+++ initial/file2	2020-11-18 16:06:12.012873364 +0000
@@ -1,3 +1,2 @@
 sdmsdpc
-ksdsdckds
-vdkndskl
\ No newline at end of file
+ksdsdckds
\ No newline at end of file
