diff -ruN '--exclude=.lit' /home/ec2-user/lit/testdir/.lit/initial_copy/subfolder/file2 /home/ec2-user/lit/testdir/subfolder/file2
--- /home/ec2-user/lit/testdir/.lit/initial_copy/subfolder/file2	1970-01-01 00:00:00.000000000 +0000
+++ /home/ec2-user/lit/testdir/subfolder/file2	2020-11-19 11:36:46.874232186 +0000
@@ -0,0 +1,2 @@
+This is the first line of the second file.
+And another line in the second file.
